project/package.scala

```diff
@@ -0,0 +1,7 @@
+package object akka {
+
+  implicit class RichBoolean(val b: Boolean) extends AnyVal {
+    final def apply[A](a: => A): Option[A] = if (b) Some(a) else None
+  }
```

Comment:
It reduces boilerplate when used together with boolean flags that we have quite a few. Therefore it is not possible to add it to a narrower scope.

For example previously we had:
```
if (CliOption.sbtLogNoFormat) List("-Dakka.test.nocolor=true") else Nil
```
And now we have instead:
```
CliOption.sbtLogNoFormat("-Dakka.test.nocolor=true").toList
```

WDYT?